packages/ocaml_intrinsics/ocaml_intrinsics.v0.15.0/opam

```diff
@@ -14,6 +14,7 @@ depends: [
   "dune"              {>= "2.0.0"}
   "dune-configurator"
 ]
+available: (arch = "x86_32" | arch = "x86_64") & os != "win32"
```

Comment:
This may need further restrictions, since `ocaml_intrinsics` doesn't seem to work on macOS: see https://github.com/janestreet/ocaml_intrinsics/issues/4